UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -818,27 +857,14 @@ private void AshBodyPart()
 					var organ = item.ItemObject?.GetComponent<BodyPart>();
 					if (organ != null)
 					{
-						if (organ.DeathOnRemoval)
+						if (organ.gibsEntireBodyOnRemoval)
```

Comment:
is this a public variable/property in camelCase?

Reply:
no it should be private, fixing this.